libraries/streaming-e2e-tests/bot/.env

```diff
@@ -0,0 +1,4 @@
+MicrosoftAppId=
+MicrosoftAppPassword=
+DIRECT_LINE_SECRET=
```

Comment:
Add `DIRECT_LINE_URL` here?

Reply:
n/a now that we are only going to request token from the DL ASE token gen endpoint